whylog/tests/tests_constraints/__init__.py

```diff
@@ -0,0 +1,96 @@
+from unittest import TestCase
+
+from whylog.config.investigation_plan import Clue, LineSource
+from whylog.constraints.verifier import InvestigationResult, Verifier
+from whylog.front import FrontInput
+
+
+class TestBasic(TestCase):
```

Comment:
I feel lack of test that takes clues dict which is produced by SearcherManager and some rules with constraint and return some InvestigationResults. Now is time for more functional tests